### PR TITLE
Update tutorial.md

### PR DESCRIPTION
**Pull request by dharmit**
**Description of the change:**
Fixes a typo

**Motivation for the change:**

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: website/content/en/docs/building-operators/golang/tutorial.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ This will scaffold the Memcached resource API at `api/v1alpha1/memcached_types.g
 
 For an in-depth explanation of Kubernetes APIs and the group-version-kind model, check out these [kubebuilder docs][kb-doc-gkvs].
 
-In general, it's recommended to have one controller responsible for manage each API created for the project to
+In general, it's recommended to have one controller responsible to manage each API created for the project to
```
Comment (varshaprasad96) [inline website/content/en/docs/building-operators/golang/tutorial.md:100]: Nit:
```suggestion
In general, it's recommended to have one controller responsible for managing each API created for the project to
```

**Comment by jberkhahn** [comment]
looks like you need to sign your commit (with the -s option)

**Review by jmrodri — APPROVED** [review]
/lgtm

**Review by rashmigottipati — APPROVED** [review]
/lgtm

**Pull request merged**